api/src/pcapi/alembic/versions/20210603_a5c58daf6917_create_table_offer_sub_category.py

```diff
@@ -26,7 +27,7 @@ def upgrade():
         sa.Column("isEvent", sa.Boolean(), nullable=False),
         sa.Column("proLabel", sa.Text(), nullable=False),
         sa.Column("appLabel", sa.Text(), nullable=False),
-        sa.Column("conditionalFields", sa.ARRAY(sa.Text()), nullable=True),
+        sa.Column("conditionalFields", postgresql.ARRAY(sa.Text()), nullable=True),
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏Ça ne demande pas de faire une migration "update" pour le format de,la colonne ?

Reply:
Non car a priori c'est mappé sur le même type postgre : `text[]`
cf [la doc postgre](https://www.postgresql.org/docs/current/arrays.html#ARRAYS-DECLARATION)